js/react_native/android/src/main/java/ai/onnxruntime/reactnative/OnnxruntimeExtensionsDisabled.java

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package ai.onnxruntime.reactnative;
+
+import ai.onnxruntime.OrtSession.SessionOptions;
+import android.util.Log;
+
+class OnnxruntimeExtensions {
+  public void registerOrtExtensionsIfEnabled(SessionOptions sessionOptions) {
+    Log.d("Info", "ORT Extensions is not enabled.");
```

Comment:
nit: Can we update the message to tell them how to enable it as clearly they're wanting to?